client/src/components/overview/QuantitySelector.jsx

```diff
@@ -40,34 +66,57 @@ const QuantitySelector = (props) => {
     return quantities;
   }
 
-  const selectAttributes = getAttributes(quantitiesAvailable);
-  const quantities = getQuantities(quantitiesAvailable);
-  if (quantitiesAvailable) {
-    resetToDefaultSelected();
+  onChange(e) {
+    const newQuantityValue = e?.target?.value;
```

Comment:
Is it possible to get an onChange event without a target? Why null-check these?

Reply:
I don't know about in normal usage, but in unit testing when these components were shallow rendered or called with incomplete props, I'd get undefined this far down when I wasn't explicitly checking the values. Maybe not the best way to deal with it, but the null checks turned off erroneous errors for testing - and I figure if it is undefined at one point, better to result in undefined than breaking the app.